### PR TITLE
Ja localization

### PR DESCRIPTION
**Pull request by Mayuko-Sa**
Updated EN -> JA documentation translations

- English sentences exist in this PR. Please leave them as they are. (They will be translated next time.)
- Some URLs of the docs do not have `/ja`. Please leave them as they are. (They will be handled in another PR.)

### CODE REVIEW

File: jekyll/_cci2_ja/caching.md [hunk 1]
```diff
@@ -331,7 +331,7 @@ myapp-+KlBebDceJh_zOWQIAJDLEkdkKoeldAldkaKiallQ=
...
-| {% raw %}`{{ .Environment.variableName }}`{% endraw %}{:.env_var} | 環境変数 `variableName` ([CircleCI からエクスポートされる環境変数](https://circleci.com/ja/docs/2.0/env-vars/#circleci-environment-variable-descriptions)、または特定の[コンテキスト](https://circleci.com/ja/docs/2.0/contexts)に追加した環境変数がサポートされ、任意の環境変数は使用できません)。                                                                                                                                |
+| {% raw %}`{{ .Environment.variableName }}`{% endraw %}{:.env_var} | 環境変数 `variableName` ([CircleCI からエクスポートされた環境変数]({{site.baseurl}}/ja/2.0/env-vars/#circleci-environment-variable-descriptions)、または特定の[コンテキスト]({{site.baseurl}}/ja/2.0/contexts)に追加された環境変数がサポートされ、任意の環境変数は使用できません)。                                                                                                                                                 |
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/caching.md:334]: just noticed the extra whitespace

```suggestion
| {% raw %}`{{ .Environment.variableName }}`{% endraw %}{:.env_var} | 環境変数 `variableName` ([CircleCI からエクスポートされた環境変数]({{site.baseurl}}/ja/2.0/env-vars/#circleci-environment-variable-descriptions)、または特定の[コンテキスト]({{site.baseurl}}/ja/2.0/contexts)に追加された環境変数がサポートされ、任意の環境変数は使用できません)。                                                                                                                    |
```

File: jekyll/_cci2_ja/java-oom.md [hunk 2]
```diff
@@ -15,15 +15,15 @@ CircleCI で Java メモリ エラーを回避およびデバッグする方法
 
 [Java 仮想マシン](https://ja.wikipedia.org/wiki/Java仮想マシン) (JVM) は、Java ベースのアプリケーションに移植可能な実行環境を提供します。 メモリ制限が設定されていない場合、JVM はシステムで使用可能な合計メモリの一部を事前に割り当てます。 CircleCI は大量のメモリを搭載した大規模なマシンでコンテナ ベースのビルドを実行しており、 各コンテナには、マシンで使用可能な総量よりも少ない量のメモリ制限が設定されています。 こうしたことから、JVM がマシン上の大量のメモリを使用可能であると認識して、コンテナに割り当てられているよりも多くのメモリを使用しようとすることがあります。
 
-By default, Java's is configured so that it will use:
-- More than `1/64th` of your total memory (for Docker Medium with 4GiB of RAM this will be 64 MiB)
-- Less than `1/4th` of your total memory (for Docker Medium with 4GiB of RAM this will be 1GiB).
+デフォルトでは、Java の使用量は以下のように設定されています。
+- 合計メモリの `1/64` 以上（4 Gib の RAM の Docker Medium クラスの場合、64 MiB）
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/java-oom.md:19]:
```suggestion
- 合計メモリの `1/64` 以上（4 GiB の RAM の Docker Medium クラスの場合、64 MiB）
```

File: jekyll/_cci2_ja/java-oom.md [hunk 3]
```diff
@@ -15,15 +15,15 @@ CircleCI で Java メモリ エラーを回避およびデバッグする方法
...
+- 合計メモリの `1/4` 以下（4 Gib の RAM の Docker Medium クラスの場合、１ GiB）
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/java-oom.md:20]:
```suggestion
- 合計メモリの `1/4` 以下（4 GiB の RAM の Docker Medium クラスの場合、１ GiB）
```

File: jekyll/_cci2_ja/server-3-install-build-services.adoc [hunk 4]
```diff
@@ -345,7 +345,7 @@ endif::env-aws[]
 
 ==== 設定とデプロイ
 
-Nomad クライアントの導入が完了したら、 CircleCI Server と Nomad コントロールプレーンを設定できます。 KOTS の管理コンソールにアクセスします。 名前空間を変更して `kubectl kots admin-console -n <YOUR_CIRCLECI_NAMESPACE>` を実行し、KOTS 管理者コンソールにアクセスします。
+Nomad クライアントの導入が完了したら、 CircleCI Server と Nomad コントロールプレーンを設定できます。 名前空間を変更して `kubectl kots admin-console -n<YOUR_CIRCLECI_NAMESPACE>`を実行し、KOTS 管理者コンソールにアクセスします。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/server-3-install-build-services.adoc:348]:
```suggestion
Nomad クライアントの導入が完了したら、 CircleCI Server と Nomad コントロールプレーンを設定できます。 名前空間を変更して `kubectl kots admin-console -n <YOUR_CIRCLECI_NAMESPACE>`を実行し、KOTS 管理者コンソールにアクセスします。
```

File: jekyll/_cci2_ja/server-3-install-build-services.adoc [hunk 5]
```diff
@@ -732,29 +732,28 @@ NOTE: 静的 JSON 認証情報から Workload Identity に切り替える場合
...
+VM サービスを KOTS 管理者コンソールから設定します。 利用可能な設定オプションの詳細については、 https://circleci.com/docs/2.0/server-3-operator-vm-service[VM サービス]ガイドを参照してください。
 +
 フィールドの設定が完了したら、*設定を保存し*、更新したアプリケーションをデプロイします。
 
 endif::env-aws[]
 
 ==== VM サービスの検証
 
-CircleCI Server の設定が完了したら、VM サービスが適切に動作しているか確認する必要がありあます。 CircleCI Server 内で、realitycheckプロジェクトを再実行できます。 緑色であれば VM サービスジョブは完了してます。 この時点で、すべてのテストが緑色で合格しているはずです。
+CircleCI Server の設定とデプロイが完了したら、VM サービスが適切に動作しているか確認する必要がありあます。 CircleCI Server 内で、realitychecker プロジェクトを再実行できます。 VM サービスジョブは完了しているはずです。 この時点で、すべてのテストが合格しているはずです。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/server-3-install-build-services.adoc:743]: the project name is realitycheck, as per the original text 🤓 
https://github.com/circleci/realitycheck

```suggestion
CircleCI Server の設定とデプロイが完了したら、VM サービスが適切に動作しているか確認する必要がありあます。 CircleCI Server 内で、realitycheck プロジェクトを再実行できます。 VM サービスジョブは完了しているはずです。 この時点で、すべてのテストが合格しているはずです。
```

File: jekyll/_cci2_ja/server-3-install-hardening-your-cluster.adoc [hunk 6]
```diff
@@ -78,23 +78,23 @@ NOTE: Server v3.3.0 以降は、リバースプロキシが https://github.com/t
...
 ### 送信
-ほとんどの環境では、インスタンスすべてに対してインターネット リソースへのアクセスを許可することになります。 これを行うには、VPC 内にある DNS サーバーへのポート 53 からの UDP と TCP の送信を許可する必要があります。また、HTTP と HTTPS それぞれのトラフィックについて、ポート 80 と 443 からの TCP の送信を許可する必要もあります。
+ほとんどの環境では、インスタンスすべてに対してインターネットリソースへのアクセスを許可することになります。 これを行うには、VPC 内にある DNS サーバーへのポート 53 からの UDP と TCP の送信を許可する必要があります。また、HTTP と HTTPS それぞれのトラフィックについて、ポート 80 と 443 からの TCP の送信を許可する必要もあります。
 ジョブのビルドを行うインスタンス (Nomad クライアントや外部 VM) では、多くの場合、SSH 経由でご利用中の VCS からコードをプルする必要があります (TCP ポート 22)。 SSH は、外部 VM との通信にも使用されます。 そのため、少なくとも送信先が VM サブネットおよび VCS であるインスタンスすべてについて、SSH を許可する必要があります。
 
 ## Kubernetes ノード
 
 ### ノード間のトラフィック
-デフォルトでは、K8s クラスタ内のトラフィックはネットワーク ポリシーにより制限されています。 つまり、K8s ノード間のトラフィックを特別に制限する必要はなく、これらノード間のトラフィックはすべて許可してかまいません。
+デフォルトでは、Kuberntes クラスタ内のトラフィックはネットワークポリシーにより規定されています。 つまり、Kubernetes ノード間のトラフィックを特別に制限する必要はなく、Kubernetes ノード間のトラフィックはすべて許可してかまいません。
```
Comment (kelvintaywl) [inline jekyll/_cci2_ja/server-3-install-hardening-your-cluster.adoc:97]:
```suggestion
デフォルトでは、Kubernetes クラスタ内のトラフィックはネットワークポリシーにより規定されています。 つまり、Kubernetes ノード間のトラフィックを特別に制限する必要はなく、Kubernetes ノード間のトラフィックはすべて許可してかまいません。
```

**Review by kelvintaywl — APPROVED** [review]

**Review by kbatuigas — APPROVED** [review]

**Pull request merged**